core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.html

```diff
@@ -54,7 +54,9 @@
         </button>
       </td>
       <td class="delete-icon">
-        <span ng-if="canDeleteSkill()" ng-click="deleteSkill(skill.id)" class="fas fa-trash-alt list-summary protractor-test-delete-skill-button"> </span>
+        <a ng-if="canDeleteSkill()" ng-click="deleteSkill(skill.id)" class="fas fa-trash-alt list-summary protractor-test-delete-skill-button">
+          <span></span>
```

Comment:
Move the icon related class inside the span class. (The way you have done above^)

Reply:
(Also, should we use `<i>` tag to render icons as [suggested by font-awesome](https://fontawesome.com/v4.7.0/icon/trash)? what do you think?)

Reply:
I was looking into the usage of `i` tag, and in this [link](https://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span), it is criticized. Some users do say that it might be better due to some reasons, but I guess for our use, span might still be better (semantically). Refer to the link for a better insight.
So I guess the final code would look like:
```<span class="fas fa-trash-alt list-summary"></span>```

Reply:
Thanks for looking into this, I agree with you, let's use `span` tag.

Reply:
Done